sound/soc/intel/common/soc-acpi-intel-jsl-match.c

```diff
@@ -10,6 +10,13 @@
 #include <sound/soc-acpi-intel-match.h>
 
 struct snd_soc_acpi_mach snd_soc_acpi_intel_jsl_machines[] = {
+	{
+		.id = "DLGS7219",
+		.drv_name = "jsl_da7219_max98373",
+		.machine_quirk = snd_soc_acpi_codec_list,
```

Comment:
is this needed if you don't have a .quirk_data?

Reply:
Ack, will check.